SnakeGame/view.py

```diff
@@ -1,14 +1,51 @@
-import pygame
+import pygameimport pygame
+from model import CellContent
+import enum
 
 class GameView(object):
     
-    DEFAULT_SIZE = width, length = 540, 540
     BG_COLOR = green = 0, 135, 0
+    SNAKE_COLOUR = 135, 0, 0
+    FOOD_COLOUR = 255, 51, 0
+    SNAKEHEAD_COLOR = 0, 0, 0
+    SNAKEHEAD_OFFSET = 0.1
+    SNAKEBODY_OFFSET = 0.05
+    FOOD_OFFSET = 0.25
 
-    def __init__(self):
-        self.size = self.DEFAULT_SIZE
+    def __init__(self, gridDimensions, windowSize=(540, 540)):
+        #Grid dimensions in number of cells
+        self.width = gridDimensions[0]
+        self.height = gridDimensions[1]
+        #grid dimensions in pixels
+        self.windowSize=windowSize
+        #grid cell dimensions in pixels
+        self.cellWidth = windowSize[0]/self.width
+        self.cellHeight = windowSize[1]/self.height
 
-    def init(self):
-        self.screen = pygame.display.set_mode(self.size)
+    def render(self, grid):
+        self.screen = pygame.display.set_mode(self.windowSize)
         self.screen.fill(self.BG_COLOR)
-        pygame.display.flip()
+        
+        for i in range(self.height):
+            for j in range(self.width):
+                if grid[i][j] == CellContent.SnakeBody:
+                    pygame.draw.rect(self.screen, self.SNAKE_COLOUR, (self.getleftCoord(j, self.SNAKEBODY_OFFSET), self.getTopCoord(i, self.SNAKEBODY_OFFSET), self.cellWidth*(1 - 2.0*self.SNAKEBODY_OFFSET), self.cellHeight*(1 - 2.0*self.SNAKEBODY_OFFSET)))
+                elif grid[i][j] == CellContent.SnakeHead:
+                     pygame.draw.rect(self.screen, self.SNAKEHEAD_COLOR, (self.getleftCoord(j, self.SNAKEHEAD_OFFSET), self.getTopCoord(i, self.SNAKEHEAD_OFFSET), self.cellWidth*(1 - 2.0*self.SNAKEHEAD_OFFSET), self.cellHeight*(1 - 2.0*self.SNAKEHEAD_OFFSET)))
+                elif grid[i][j] == CellContent.Food:
+                    pygame.draw.ellipse(self.screen, self.FOOD_COLOUR, (self.getleftCoord(j, self.FOOD_OFFSET), self.getTopCoord(i, self.FOOD_OFFSET), (self.cellWidth)/2.0, (self.cellHeight)/2.0))
+        pygame.display.flip()          
```

Comment:
pygame keeps printing DeprecationWarning as its draw functions apparently expect integers only. Super weird. I'll add a flag in the continuous-integration branch to suppress the warning (it doesn't seem worth investing a lot of effort into considering that the primary objective of this project goes well beyond the game itself.